src/integration-test/tests/legal/home/steps/user.ts

```diff
@@ -1,90 +1,110 @@
-'use strict'
-
 const DEFAULT_PASSWORD = require('../../../../data/test-data').DEFAULT_PASSWORD
 
-let loginPage, startClaimPage, yourOrganisationNamePage, yourOrganisationAddressPage, yourContactDetailsPage,
-  yourReferencePage, yourCountyCourtPage, claimantTypePage, claimantAddressPage, claimantAddPage
-
-module.exports = {
-
-  _init () {
-    loginPage = require('../pages/login')
-    startClaimPage = require('../../claim/pages/claimant-start-claim')
-    yourOrganisationNamePage = require('../../claim/pages/claimant-representative-name')
-    yourOrganisationAddressPage = require('../../claim/pages/claimant-representative-address')
-    yourContactDetailsPage = require('../../claim/pages/claimant-representative-contacts')
-    yourReferencePage = require('../../claim/pages/claimant-reference')
-    yourCountyCourtPage = require('../../claim/pages/claimant-preferred-court')
-    claimantTypePage = require('../../claim/pages/claimant-type')
-    claimantAddressPage = require('../../claim/pages/claimant-address')
-    claimantAddPage = require('../../claim/pages/claimant-add')
-  },
+import { LoginPage } from '../pages/login'
+import { ClaimantStartClaimPage } from '../../claim/pages/claimant-start-claim'
```

Comment:
We are preferring to avoid relative paths. You can add a path to `claim` in `tsconfig.json` and then search on the inspection. Check that there isn't an issue in the backlog already to tackle that technical debt.

Reply:
good spot, I think that can be done as a part of the other story

Reply:
I think since we are developing new code we should address the issue new. Especially that `integration-tests` TS alias should be already there

Reply:
In other places we just have `integration-test/...`